src/main/java/com/fundacionjala/pivotalapi/RequestManager.java

```diff
@@ -0,0 +1,58 @@
+package com.fundacionjala.pivotalapi;
+
+import java.util.Map;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.response.Response;
+import com.jayway.restassured.specification.RequestSpecification;
+import org.apache.log4j.Logger;
+
+import static com.jayway.restassured.RestAssured.given;
+
+
+public final class RequestManager {
+
+    private static final Logger LOGGER = Logger.getLogger(PropertiesInfo.class);
+
+    private static final RequestSpecification REQUEST_SPECIFICATION = Authentication.getInstance().getRequestSpecification();
+
+    private RequestManager() {
+    }
+
+    public static Response getRequest(String endpoint) {
+        //    System.out.println(endpoint);
+        LOGGER.info("GET :" + endpoint);
+        return REQUEST_SPECIFICATION.when().get(endpoint);
+    }
+
+    public static Response postRequest(String endpoint, Map<String, Object> parameters) {
+        LOGGER.info("POST :" + endpoint);
+        JsonElement jsonObject = new JsonParser().parse(parameters.toString());
```

Comment:
Instead of converting to JsonElement you can send it directly like params
Please ask to the team